app/liquid/liquid_file_system.rb

```diff
@@ -1,20 +1,61 @@
+# The +LiquidFileSystem+ class is used by +Liquid+ for
```

Comment:
`<tt>..</tt>` and `+..+` denotes fixed width font.

http://guides.rubyonrails.org/api_documentation_guidelines.html